Shared/Samples/Identify features in WMS layer/IdentifyFeaturesInWMSLayerView.swift

```
@@ -0,0 +1,112 @@
+        let center = Point(x: -95.7129, y: 37.0902, spatialReference: .wgs84)
+        map.initialViewpoint = Viewpoint(center: center, scale: 7e7)
+        return map
+    }()
+    
+    /// The WMS layer with EPA water info.
+    @State private var waterInfoLayer: WMSLayer?
+    
+    /// The text of a WMS feature HTML attribute that is shown in the web view.
+    @State private var webViewText = String()
+
+    /// The tapped screen point.
+    @State var tapScreenPoint: CGPoint?
+    
+    /// The string text for the identify layer results overlay.
+    @State var overlayText = "Tap on the map to identify features in the WMS layer."
+
+    /// The error shown in the error alert.
+    @State private var error: Error?
+
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map)
+                .onSingleTapGesture { screenPoint, _ in
+                    tapScreenPoint = screenPoint
+                }
+                .task {
+                    do {
+                        // Create a WMS layer from a URL and load it.
+                        let wmsLayer = WMSLayer(url: .EPAWaterInfo, layerNames: ["4"])
+                        try await wmsLayer.load()
+                        
+                        // Add the layer to the map.
+                        map.addOperationalLayer(wmsLayer)
+                        
+                        waterInfoLayer = wmsLayer
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .task(id: tapScreenPoint) {
+                    // Identify on WMS layer using the screen point.
+                    webViewText = if let screenPoint = tapScreenPoint,
+                                     let waterInfoLayer,
+                                     // Identify feature on water info layer
+                                     let identifyResult = try? await mapViewProxy.identify(
```

Since you are already using an error alert in this sample, would it make sense to catch the error from the identify instead of failing silently?